### PR TITLE
Enable Less and SCSS highlighting

### PR DESCRIPTION
**Pull request by walenok**
Solving issue #1367 

Added support for .less and .scss extensions enabling additional opetions for the CSS highlighter

### CODE REVIEW

File: PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp [hunk 1]
```diff
@@ -633,6 +635,15 @@ void ScintillaEditView::setJsonLexer()
 	execute(SCI_SETPROPERTY, reinterpret_cast<WPARAM>("fold.preprocessor"), reinterpret_cast<LPARAM>("1"));
 }
 
+void ScintillaEditView::setCssLexer(LangType type) 
+{
+	setLexer(SCLEX_CSS, L_CSS, LIST_0 | LIST_1);
+	if (type == L_LESS)
```
Comment (dail8859) [inline PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp]: Should this just be `if (type == L_LESS || type == L_SCSS)` or should they be setting different properties for each?

**Comment by dail8859** [comment]
There should also be an update to `stylers.model.xml` for the styling information?

File: PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp (hunk 1 above)
Comment (walenok) [inline PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp]: Yes, L_SCSS should set property lexer.css.scss.language.
My bad.

**Comment by EnigmaSolved** [comment]
You may already have everything you need, but if you're needing any additional styling particular to SCSS, I've been using the following for several years and it has worked well:
https://github.com/marvinlabs/notepad-plus-plus-scss-syntax-highlighting

File: PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp [hunk 2]
```diff
@@ -139,6 +139,8 @@ LanguageName ScintillaEditView::langNames[L_EXTERNAL+1] = {
 {TEXT("json"),			TEXT("json"),				TEXT("JSON file"),										L_JSON,			SCLEX_CPP },
 {TEXT("javascript.js"), TEXT("JavaScript"),			TEXT("JavaScript file"),								L_JAVASCRIPT,	SCLEX_CPP },
 {TEXT("fortran77"),		TEXT("Fortran fixed form"),	TEXT("Fortran fixed form source file"),					L_FORTRAN_77,	SCLEX_F77},
+{TEXT("less"),			TEXT("Less"),				TEXT("Less File"),                                      L_LESS,			SCLEX_CSS },
+{TEXT("scss"),			TEXT("SCSS"),				TEXT("SASS File"),						                L_SCSS,			SCLEX_CSS },
```
Comment (MAPJe71) [inline PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp]: Shouldn't it be "S**C**SS File" or "Sassy CSS File" as SASS syntax is not equal to SCSS syntax?

Comment (NN---) [inline PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp]: SASS is indentation based.
SCSS is superset of CSS.
https://responsivedesign.is/articles/difference-between-sass-and-scss

**Pull request closed without merging**